app/models/role.rb

```diff
@@ -3,10 +3,10 @@
 class Role < ApplicationRecord
   validates :group, inclusion: [Person::STUDENT, Person::MENTOR]
   validates :title, presence: true
-  validates_uniqueness_of :title, scope: :organization_id
-  belongs_to :organization
+  validates_uniqueness_of :title, scope: :team_id
+  belongs_to :team
   has_many :people
```

Comment:
Volgens mij moet hier ook een dependent destroy achter. `role_id` mag namelijk niet null zijn in de database volgens de constraints. Dus wat doe je anders als je een role delete? Dan krijg je volgens mij of een error, of een id van een role die niet meer bestaat (of geeft dat ook een error?)

Het is wel idd zo van je kan niet zomaar even een role deleten, maar dat wil je ook niet zomaar doen. En als je dat wel doet dan is het wel logisch dat iedereen die die role heeft ook weg gaat. Anders als je een organization namelijk delete, dan blijven er allemaal person objecten achter in de database met role ids die niet meer kloppen (en dan gaan volgens mij ook sommige mentor checks fout dus dit willen we helemaal niet).

Reply:
hmm, willen we dat inderdaad ja? ergens vind ik het wel spannend dat bij het verwijderen van een role alle personen die eronder hangen ook weg gegooid worden. Aan de andere kant geldt dat voor de meeste dependent destroys. Ik voeg m wel toe, maar idd, we moeten bij het mergen / deployen van deze changes eerst uitgebreid lokaal checken of alles goed gaat.